python/lsst/qserv/schema/migrations/worker/migrate-2-to-3.sql

```diff
@@ -9,11 +9,11 @@ DROP USER IF EXISTS 'monitor'@'%';
 -- Remove superfluous grants from former admin module
 -- -----------------------------------------------------
 
-REVOKE ALL PRIVILEGES ON `qservMeta`.* FROM 'qsmaster'@'localhost';
-REVOKE ALL PRIVILEGES ON `qservMeta`.* FROM 'qsmaster'@'%';
+REVOKE IF EXISTS ALL PRIVILEGES ON `qservMeta`.* FROM 'qsmaster'@'localhost';
+REVOKE IF EXISTS ALL PRIVILEGES ON `qservMeta`.* FROM 'qsmaster'@'%';
 
-REVOKE ALL PRIVILEGES ON `qservCssData`.* FROM 'qsmaster'@'localhost';
-REVOKE ALL PRIVILEGES ON `qservCssData`.* FROM 'qsmaster'@'%';
+REVOKE IF EXISTS ALL PRIVILEGES ON `qservCssData`.* FROM 'qsmaster'@'localhost';
+REVOKE IF EXISTS ALL PRIVILEGES ON `qservCssData`.* FROM 'qsmaster'@'%';
 
-REVOKE ALL PRIVILEGES ON `qservResult`.* FROM 'qsmaster'@'localhost';
-REVOKE ALL PRIVILEGES ON `qservResult`.* FROM 'qsmaster'@'%';
+REVOKE IF EXISTS ALL PRIVILEGES ON `qservResult`.* FROM 'qsmaster'@'localhost';
+REVOKE IF EXISTS ALL PRIVILEGES ON `qservResult`.* FROM 'qsmaster'@'%';
```

Comment:
I have tested this in MariaDB, and sadly, `IF EXISTS` is not supported by MariaDB: https://mariadb.com/docs/server/reference/sql-statements/account-management-sql-statements/revoke. On the other hand, MySQL does support this: https://dev.mysql.com/doc/refman/8.4/en/revoke.html.

So, we need to find some other way to address this issue. Perhaps, we should remove those entries from the very origin (where the were added to the schema definition files) or simply forget about those?